app/assets/javascripts/search.js

```diff
@@ -16,7 +16,7 @@
       search.trackSearchResultsAndSuggestionsOnPageTrack($searchResults);
     },
     buildSearchResultsData: function ($searchResults) {
-      var searchResultData = {'urls': []},
+      var searchResultData = {'urls': [], 'suggestion': ""},
```

Comment:
not sure you need this change. You will still be able to call  `if(searchResultData.suggestion)` without it.